data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,1002 @@
+		conversation
+			`When you return, the Hai Captain thanks you profusely. "The Allhome government has authorized a reward of <payment> for helping to route the pirate problem," he explains while handing you the credits.`
+			branch "saved geocoris"
+				has "00 Hai Leaks Response 4B: declined"
+			`	He sighs deeply. "It is a shame though that the kidnapped Hai could not be saved, but the other Hai captains tell me it was a mess out there."`
+			`	In an oddly human gesture that feels very deliberate, he places one of his furry hands comfortingly on your shoulder, "I'm sure you did all you could."`
+				goto end
+			label "saved geocoris"
+			`	"Your rescue of our kidnapped people is deeply appreciated also. Lucky for you, the merchant who owned the Geocoris has already filed the insurance claim, so you get to keep the ship as a trophy."`
+			`	"I do need to do a full security scan of it to complete all the incident paperwork before I let it go, so if you have any business on-world here, please take your time."`
+			label end
+			`	"In any event, let's hope a more permanent solution can be devised soon. In the meantime, the local dealers have decided to start selling the Ladybugs again. I hear it's something about a spike in interest among the humans here, so feel free to have a look at them. I shouldn't get too distracted though, I'm sure they'll be waiting for you back on Hai-home."`
+			branch "saved geocoris2"
+				has "00 Hai Leaks Response 4B: declined"
+			`	With a final nod of thanks, he departs to attend to his business: leaving you free to go about your own.`
+				decline
+			label "saved geocoris2"
+			`	With a final nod of thanks, he departs to attend to his business: leaving you free to wander the spaceport for a couple of hours until they release your new ship.`
+				decline			
+	
+event "waypoint pirates vanquished"
+	#Update pirate and anti-pirate spawns in Waypoint and Heia Due
+	system "Waypoint"
+		government "Hai"
+		add fleet "Anti-Pirate Hai (Wormhole Travel)" 1000
+		remove fleet "Marauder fleet IX"
+		remove fleet "Anti-Pirate Hai (No Wormhole Travel)"
+		remove fleet "Large Northern Pirates"
+		remove fleet "Large Core Pirates"
+	system "Heia Due"
+		remove fleet "Marauder fleet IX"
+		remove fleet "Large Northern Pirates"
+		remove fleet "Large Core Pirates"
+
+# It is impossible to implement a "capture and return npc" mission. 
+# Once the ship is captured, the NPC is gone and the mission is failed.
+# The NPC has an outfit used to determine if you managed to get it back safely.
+mission "Hai Leaks Response 4A"
+	name "Recover the Geocoris"
+	description "Recover the stolen Geocoris supporting the pirates in Waypoint, and return to Allhome. (You will need to swap the Geocoris to your flagship, and depart and land again in order to progress the missions properly.)"
```

Comment:
This is definitely not mergeable in this state. We'll either need to rework these missions to get the intended effect (I'd need to sit and think about it for a while to see if this is possible using existing mechanics), wait on new mechanics, or ditch the concept of capturing the ship; perhaps we just use a board conversation instead.

Reply:
All it needs is for #4561 to get resurrected and finished - and be able to properly register outfits on captured ships immediately when you first land with them.

Reply:
"It works," isn't sufficient enough to be mergeable.

Reply:
~After 2.5 years, with the PR that was meant to fix the issue being shelved, maybe it ought to be.~